### ISSUE TITLE
Add Unreal Engine

### ISSUE THREAD

**Issue by FlamingPhoenix2YT** [leereilly/games#383]

**Comment by Translator5** [comment]
added in #359

**Comment by Translator5** [comment]
It is private, and not viewable without permissions!

**Comment by NLDev** [comment]
> It is private, and not viewable without permissions!

It is not really private. Anyone is free to join 
https://github.com/EpicGames/Signup

There is no approving process and no permissions are needed. 
Once signed up, you can fork, clone and view all the code.

**Comment by xDShot** [comment]
No. EULA says it's not publically available. And signing up process for everyone doesn't make it public.

**Comment by Luraktinus** [comment]
its does not belong here
